examples/cron-backfill.yaml

```diff
@@ -0,0 +1,114 @@
+# This is an example of back-filling day for a daily cron workflow.
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: job
+spec:
+  arguments:
+    parameters:
+      - name: date
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: date
+            value: "{{inputs.parameters.date}}"
+            # use "yesterday" is a magic value to tell the script it needs to determine the day to run
+            default: yesterday
+      script:
+        image: docker/whalesay:latest
+        command:
+          - sh
+        source: |
+          date="{{inputs.parameters.date}}"
+          if [ $date = yesterday ]; then
+            date=$(date -d yesterday +%Y-%m-%d)
+          fi
```

Comment:
Shouldn't the default action here (i.e. if no inputs are passed in) to do nothing?

Reply:
no, I think by default it needs to run as if it was a normal daily cron

Reply:
@alexec the argo list shows an error on this, can you point me to the updated one if there is any?